convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -980,6 +980,104 @@ def test_get_packages_to_update_dnf(packages, reposdir, pretend_os, monkeypatch)
     assert _get_packages_to_update_dnf(reposdir=reposdir) == packages
 
 
+@pytest.mark.skipif(
+    pkgmanager.TYPE != "dnf",
+    reason="No dnf module detected on the system, skipping it.",
+)
+def test_get_packages_to_update_dnf_rhel_repos(monkeypatch):
+    # Mock the uneccesary calls for testing
+    monkeypatch.setattr(pkgmanager.Base, "fill_sack", value=mock.Mock())
+    monkeypatch.setattr(pkgmanager.Base, "upgrade_all", value=mock.Mock())
+    monkeypatch.setattr(pkgmanager.Base, "resolve", value=mock.Mock())
+    monkeypatch.setattr(pkgmanager.Base, "transaction", value=[])
+
+    rhel_repo_id = "rhel-8-for-x86_64-baseos-rpms"
+
+    # Create DNF Base object
+    base = pkgmanager.Base()
+
+    # Add RHEL repo to the DNF Base
+    base.repos.add_new_repo(rhel_repo_id, base.conf, baseurl="https://random.testing.url/")
+
+    # Get all enabled repos in the current system
+    base.read_all_repos()
+    enabled_repos = [repo.id for repo in base.repos.iter_enabled()]
+
+    # Check if the list is not empty and the RHEL repo is enabled
+    assert enabled_repos
+    assert rhel_repo_id in enabled_repos
+
+    # Choose one of the enabled repos on the system
+    # Avoid getting the rhel_repo_id if on 0 index, then get the repo id from 1
```

Comment:
Why?

Reply:
I'm trying to disable one of the repos available on the system to disabling repo provided through the tool_opts (`monkeypatch.setattr(tool_opts, "enablerepo", repo_for_disable)`). I want to have something different than the added RHEL repo but still present on the system. 

So I'm getting it in this way - I know if on the 0 index is rhel repo, on the 1 will be something different I can use

Reply:
I'd write that out a bit clearer in the text, avoid getting something is clear but not super clear in the comments of why you do it

Reply:
Comment about it added.